doc/content/msr/msre/multiphysics_rz_model/msre_multiphysics_core_model.md

```diff
@@ -0,0 +1,326 @@
+The pump force was adapted to produce a loop circulation time of ~25 seconds.
+
+!listing msr/msre/multiphysics_core_model/steady_state/th.i start=# Operational Parameters end=# Thermal-Hydraulic diameters
+
+Next, the hydraulic diameters of the flow channels are set respectively. The fluid blocks are defined to indicate the fluid regions.
+
+!listing msr/msre/multiphysics_core_model/steady_state/th.i start=# Thermal-Hydraulic diameters end=# Delayed neutron
+
+Then, first section defines the delayed neutron group properties, both their production rate due to fission and decay rate. It also define the turbulent Schmidt number and the initial temperature of the fuel salt.
+
+!listing msr/msre/multiphysics_core_model/steady_state/th.i start=# Delayed neutron end=# Utils
+
+The end of the first section defines the set of blocks where the fluid flow and solid heat conduction equations will be solved.
+
+!listing msr/msre/multiphysics_core_model/steady_state/th.i start=# Utils end=# =
+
+#### Global Parameters
+
+Next, Global Parameters are defined in the `Global Parameters` block.
+This section is used to define variables and methods that are required by many methods in the input file.
+
+!listing msr/msre/multiphysics_core_model/steady_state/th.i start=[GlobalParams] end=#
+
+
+#### Mesh
+
+This block defines the geometry of the core, as shown in [MSRE_pgh_mesh_blocks]. This block reads in the same mesh that was generated using MOOSE which is identical for both Griffin and Pronghorn.
+
+!listing msr/msre/multiphysics_core_model/steady_state/th.i start=[Mesh] end=[Problem]
+
+#### Modules
+
+This block defines the Module that will be used in this problem - specifically the Navier Stokes Finite Volume action.
+
+The ```Navier Stokes Finite Volume``` action is used to define fluid properties, compute the "weakly-compressible" flow, and set up boundary conditions.
+
+Here, the action operates on the fluid blocks which are defined at the top of the header and are substituted using the `$` sign. Recall that the `$` sign refers to variable substitutions. "Weakly-compressible" with porous_medium_treatment is selected for this model and the energy equation is also included.
+
+Numerical schemes can be selected for solving the Navier Stokes equations with different methods of pressure and velocity interpolation.
+
+Porosity, friction, and turbulence treatment can be modified here using various correlations and models. Fluid properties are also set here and can be modified.
+
+The action also incorporates external physics like a volumetric heat source due to the power_density calculated by Griffin. Also, boundary conditions are set to constrain the different Navier Stokes equations.
+
+The scalar equations for delayed neutron precursor groups are not included here in the action since they are defined externally elsewhere in the input file
```

Comment:
yes but why

Reply:
This was done explicitly so that advection to be modified to account for precursors jump due to porosity.